### PR TITLE
iis: Add IIS info metric

### LINKED ISSUE #1449: [requirements]   IIS version number
There is no version number in the returned data of IIS. I see that the code has obtained the version, but there is no data to return the version number. Can this be added.

### PR DESCRIPTION
**Pull request by jkroepke**
Fixes #1449

### CODE REVIEW

**Comment by jkroepke** [comment]
Tested by https://github.com/prometheus-community/windows_exporter/issues/1449#issuecomment-2071210233

File: pkg/collector/iis/iis.go [hunk 1]
```diff
@@ -1514,6 +1531,15 @@ func (c *collector) collectW3SVC_W3WP(ctx *types.ScrapeContext, ch chan<- promet
 			continue
 		}
 
+		ch <- prometheus.MustNewConstMetric(
```
Comment (breed808) [inline pkg/collector/iis/iis.go:1534]: Is this section necessary? We appear to be adding to the channel twice for the same metric here.

File: pkg/collector/iis/iis.go [hunk 2]
```diff
@@ -1514,6 +1531,15 @@ func (c *collector) collectW3SVC_W3WP(ctx *types.ScrapeContext, ch chan<- promet
...
+			c.Info,
+			prometheus.GaugeValue,
+			app.Threads,
+			name,
+			pid,
+			"busy",
+		)
+
```
Comment (jkroepke) [inline pkg/collector/iis/iis.go:1542]:
```suggestion
```

File: pkg/collector/iis/iis.go (hunk 1 above)
Comment (jkroepke) [reply pkg/collector/iis/iis.go]: Seems like there was an mistake/leftover from copy/paste

**Review by breed808 — APPROVED** [review]
:+1:

**Pull request merged**